### PR TITLE
Mes 5130 office show me

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Description

Fixed validation on the show me question on the office screen and added functionaly to display/change outcome of questions.

## Checklist

- [x] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/modules/tests/test-data/cat-adi-part2/vehicle-checks/__tests__/vehicle-checks.cat-adi-part2.selector.spec.ts [hunk 1]
```diff
@@ -1,7 +1,37 @@
 import { CatADI2UniqueTypes } from '@dvsa/mes-test-schema/categories/ADI2';
-import { vehicleChecksExist } from '../vehicle-checks.cat-adi-part2.selector';
+import {
+  getVehicleChecksDangerous,
+  getVehicleChecksSerious,
+  vehicleChecksExist,
+} from '../vehicle-checks.cat-adi-part2.selector';
 
-describe('Vehicle Checks Selector Cat ADI2', () => {
+fdescribe('Vehicle Checks Selector Cat ADI2', () => {
```
Comment (matthew2564) [inline src/modules/tests/test-data/cat-adi-part2/vehicle-checks/__tests__/vehicle-checks.cat-adi-part2.selector.spec.ts]: remove f from describe

File: src/pages/office/cat-adi-part2/components/show-me-questions/show-me-questions.ts [hunk 2]
```diff
@@ -62,12 +75,24 @@ export class ShowMeQuestionsCatADI2Component implements OnChanges {
...
   showMeQuestionsChanged(showMeQuestions: VehicleChecksQuestion): void {
     const result: QuestionResult = {
       code: showMeQuestions.code,
       description: showMeQuestions.shortName,
+      outcome: this.questionResult.outcome,
+    };
+    this.showMeQuestionsChange.emit(result);
+  }
+
+  showMeOutcomeChanged(value): void {
```
Comment (matthew2564) [inline src/pages/office/cat-adi-part2/components/show-me-questions/show-me-questions.ts]: apply type to parameter

File: src/modules/tests/test-data/cat-adi-part2/vehicle-checks/__tests__/vehicle-checks.cat-adi-part2.selector.spec.ts [hunk 3]
```diff
@@ -1,7 +1,37 @@
...
+
+  describe('getVehicleChecksSerious', () => {
+    it('should return false if there are no serious vehicle checks recorded', ()  => {
+      const emptySerious: CatADI2UniqueTypes.VehicleChecks = { seriousFault: false };
+      const result = getVehicleChecksSerious(emptySerious);
+      expect(result).toBeFalsy();
```
Comment (Cullima) [inline src/modules/tests/test-data/cat-adi-part2/vehicle-checks/__tests__/vehicle-checks.cat-adi-part2.selector.spec.ts]: can you change these to expect().toBe(false)etc. See https://vincenttunru.com/toBeTruthy-vs-toBe-true/

File: src/pages/office/cat-adi-part2/components/show-me-questions/show-me-questions.ts (hunk 2 above)
Comment (andrewsetterfield) [reply src/pages/office/cat-adi-part2/components/show-me-questions/show-me-questions.ts]: done

File: src/modules/tests/test-data/cat-adi-part2/vehicle-checks/__tests__/vehicle-checks.cat-adi-part2.selector.spec.ts (hunk 1 above)
Comment (andrewsetterfield) [reply src/modules/tests/test-data/cat-adi-part2/vehicle-checks/__tests__/vehicle-checks.cat-adi-part2.selector.spec.ts]: removed

File: src/pages/office/cat-adi-part2/components/show-me-questions/show-me-questions.ts [hunk 4]
```diff
@@ -88,4 +113,29 @@ export class ShowMeQuestionsCatADI2Component implements OnChanges {
   findQuestion(): VehicleChecksQuestion {
     return this.questions.find(question => question.code === this.questionResult.code);
   }
+
+  evaluateLabels(): void {
```
Comment (Cullima) [inline src/pages/office/cat-adi-part2/components/show-me-questions/show-me-questions.ts]: name could perhaps be more descriptive, e.g. updateShowMeQuestionAttributes()

**Review by Cullima — APPROVED** [review]

File: src/modules/tests/test-data/cat-adi-part2/vehicle-checks/__tests__/vehicle-checks.cat-adi-part2.selector.spec.ts (hunk 3 above)
Comment (andrewsetterfield) [reply src/modules/tests/test-data/cat-adi-part2/vehicle-checks/__tests__/vehicle-checks.cat-adi-part2.selector.spec.ts]: no problem.

File: src/pages/office/cat-adi-part2/components/show-me-questions/show-me-questions.ts (hunk 4 above)
Comment (andrewsetterfield) [reply src/pages/office/cat-adi-part2/components/show-me-questions/show-me-questions.ts]: fine

**Review by matthew2564 — APPROVED** [review]

**Pull request merged**